maps/tether/tether_jobs.dm

```diff
@@ -0,0 +1,184 @@
+///////////////////////////////////
+//// Talon Jobs
+/datum/department/talon
+	name = DEPARTMENT_TALON
+	short_name = "Talon"
+	color = "#888888"
+	sorting_order = -2
+	assignable = FALSE
+	visible = FALSE
+
+/datum/job/talon/captain
+	title = "Talon Captain"
+	departments_managed = list(DEPARTMENT_TALON)
+	job_description = "The captain's job is to generate profit through trade or other means such as salvage or even privateering."
+	outfit_type = /decl/hierarchy/outfit/job/talon_captain
+
+	offmap_spawn = TRUE
+	faction = "Station" //Required for SSjob to allow people to join as it
+	departments = list(DEPARTMENT_TALON)
+	total_positions = 1
+	spawn_positions = 1
+	selection_color = "#999999"
+	economic_modifier = 7
+	minimal_player_age = 14
+	pto_type = null
+	access = list(access_talon)
+	minimal_access = list(access_talon)
```

Comment:
Might make more sense for some things like minimal_player_age, access and pto_type to be moved to general talon job definition? Same for offmap_spawn and departments.

Reply:
I had that originally. Alas, the job system cannot deal with prototype types like that without a smol rewrite I don't want to do.